DM-23162: Implement RFC-650 and state that we want to break before binary operators (#368)
lsst-dm/dm_dev_guide: python/style.rst

.. _style-guide-py-binary-operator-line-break:

Wrap lines before binary operators
----------------------------------

> `PEP 8 does not prescribe <https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator>`_ whether lines should be broken before or after binary operators.

Reviewer: ```suggestion
`PEP 8 suggests <https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator>`_ that lines should be broken before binary operators but allows after.
```
